manager/integration/tests/test_basic.py

```diff
@@ -193,8 +222,10 @@ def test_snapshot(clients, volume_name):  # NOQA
     assert volume["numberOfReplicas"] == 2
     assert volume["state"] == "detached"
 
-    volume = volume.attach(hostId=host_id)
+    lht_hostId = get_self_host_id()
+    volume = volume.attach(hostId=lht_hostId)
     volume = wait_for_volume_state(client, volume_name, "healthy")
+    volume_rw_test(volume["endpoint"])
```

Comment:
This doesn't make sense.

We don't need to test against the same type of scenario again and again.

The snapshot/backup test is to test if the data in the snapshot/backup is correct.